base/tekton.dev/pipelines/okd-coreos.yaml

```diff
@@ -111,14 +114,23 @@ spec:
         - name: ws
           workspace: shared-workspace
     - name: cosa-test
+      taskRef:
+        kind: Task
+        name: cosa-test
       params:
         - name: variant
           value: $(params.variant)
       runAfter:
         - cosa-build-baseos
+      workspaces:
+        - name: ws
+          workspace: shared-workspace
+    - name: build-layer
```

Comment:
Maybe we should run the `build-layer` task after `cosa-test` and `cosa-upload-baseos` in order to pull the freshly `baseos` built during `cosa-build-baseos` task of the current pipeline. Otherwise, we would pull the `baseos` image from the latest pipeline run